spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.feature "Visitor views a guest's show page", type: :feature do
+  scenario 'they see the guest name and list of rooms stayed in' do
+    hotel1 = Hotel.create(:hotel, name: 'The Overlook', location: "Estes Park")
+    hotel2 = Hotel.create(:hotel, name: "Stanley Hotel", location: "Estes Park")
+    room1 = Room.create(:room, suite: 'Deluxe', rate: 150, hotel: hotel1, guest: guest1)
+    room2 = Room.create(:room, suite: 'Presidential', rate: 300, hotel: hotel2, guest: guest1)
+    guest1 = Room.create(:guest, name: 'Jack Nicholson')
+    GuestRoom.create(:guest_rooms, guest: guest1, room: room1)
+    GuestRoom.create(:guest_rooms, guest: guest1, room: room2)
+
+    visit "/guests/#{guest1.id}"
+
+    expect(current_path).to eq("/guests/#{guest1.id}")
+    expect(page).to have_content('Jack Nicholson')
+    expect(page).to have_content('Deluxe')
+    expect(page).to have_content('150')
+    expect(page).to have_content('The Overlook')
+  end
```

Comment:
This test is a great start but according to the user story, we want to check that all of the guest's rooms show up when we visit their show page. Currently, your assertions only check for 1 room, the Deluxe. 

Again this is a good start to your test, we just need to add a little more to make it match the US exactly.

Beyond that, I'm also a little confused as you why you have to pass the model name as the first argument to the `create` method.

Last thing, when checking for content on a page, you want to get into the habit of checking some "anchor" text as well. This reduces the amount of ambiguity in your assertions and makes your tests more robust. For example, instead of just doing `expect(page).to have_content('150')`, you could do something like `expect(page).to have_content('Rate: $150')`. This assertion says that 150 isn't just some arbitrary number that can show up anywhere but is part of the "Rate" that I'm looking for.